Application/Source/Components/TripTable/TripTableController.swift

```diff
@@ -121,20 +121,22 @@ final class TripTableController: ControllerBase<Void, TripTableRootView>, UITabl
             .subscribe(onNext: { [reactions] _ in
                 reactions.loggedOut?()
             })
-            .addDisposableTo(lifetimeDisposeBag)
+            .disposed(by: lifetimeDisposeBag)
 
         createTripButton.rx.tap
             .flatMapLatest(reactions.createTrip)
             .subscribe()
-            .addDisposableTo(lifetimeDisposeBag)
+            .disposed(by: lifetimeDisposeBag)
 
-//        rootView.tableView.rx.setDelegate(self).addDisposableTo(lifetimeDisposeBag)
+//        rootView.rx.setDelegate(self).disposed(by: lifetimeDisposeBag)
```

Comment:
Is this line necessary? I mean this project is not new, so if it's commented out, I don't think we need it, unless you convince me otherwise, please delete it.